doc/api/cli.md

```diff
@@ -271,7 +271,7 @@ added: v4.0.0
 -->
 
 Specify an alternative default TLS cipher list. (Requires Node.js to be built
-with crypto support. (Default))
```

Comment:
The `Default` here seems somewhat confusing.

Reply:
Maybe  this?
```md
Specify an alternative default TLS cipher list. Requires Node.js to be built
with crypto support (default).
```

Reply:
Yes, that looks much better :-)